### PR TITLE
Added bridge for Nette Framework

### PR DESCRIPTION
**Pull request by dg**
see nette/bootstrap#4

Closes #58

### CODE REVIEW

File: src/Bridges/Nette/Bridge.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+							. BlueScreen::highlightFile($item['args'][0], $m[1])
+					);
+				} elseif ($item = Helpers::findTrace($e->getTrace(), 'Nette\Neon\Decoder::decode')) {
+					return array(
+						'tab' => 'NEON',
+						'panel' => BlueScreen::highlightPhp($item['args'][0], $m[1])
+					);
+				}
+			}
+		});
```
Comment (JanTvrdik) [inline src/Bridges/Nette/Bridge.php]: Why is this not in latte/latte and nette/neon?

Comment (dg) [inline src/Bridges/Nette/Bridge.php]: There are things that I do not know where to put. I hate it.

Comment (dg) [inline src/Bridges/Nette/Bridge.php]: Perhaps I'll leave it in the nette/bootstrap and solve it sometime in the future.

Comment (JanTvrdik) [inline src/Bridges/Nette/Bridge.php]: > There are things that I do not know where to put

Could you write a list of those things to https://github.com/nette/bootstrap/issues/4? Maybe someone will have a good idea.

Comment (dg) [inline src/Bridges/Nette/Bridge.php]: This file and rest of NetteExtension https://github.com/dg/nette-bootstrap/commit/c5793aa494dce77b06d61ca438b5f56e5040dc0d#diff-d4b5afc3710c208bf26e8d1fcf72f288L278

**Pull request merged**